bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java

```diff
@@ -377,4 +403,68 @@ public void onGroupGone(@Nullable HueBridge bridge, FullGroup group) {
             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.GONE, "@text/offline.group-removed");
         }
     }
+
+    /**
+     * Schedules restoration of the alert item state to {@link LightStateConverter#ALERT_MODE_NONE} after a given time.
+     * <br>
+     * Based on the initial command:
+     * <ul>
+     * <li>For {@link LightStateConverter#ALERT_MODE_SELECT} restoration will be triggered after <strong>2
+     * seconds</strong>.
+     * <li>For {@link LightStateConverter#ALERT_MODE_LONG_SELECT} restoration will be triggered after <strong>15
+     * seconds</strong>.
+     * </ul>
+     * This method also cancels any previously scheduled restoration.
+     *
+     * @param command The {@link Command} sent to the item
+     */
+    private void scheduleAlertStateRestore(Command command) {
+        cancelScheduledFuture();
+        int delay = getAlertDuration(command);
+
+        if (delay > 0) {
+            scheduledFuture = scheduler.schedule(() -> {
+                updateState(CHANNEL_ALERT, new StringType(LightStateConverter.ALERT_MODE_NONE));
+            }, delay, TimeUnit.MILLISECONDS);
+        }
+    }
+
+    /**
+     * This method will cancel previously scheduled alert item state
+     * restoration.
+     */
+    private void cancelScheduledFuture() {
```

Comment:
You should call this when disposing this handler as well.

Reply:
Yes, you're right.
I have a copy/paste from the hue light handler and this is missing in this handler too.

Reply:
Fixed